eth/v1alpha1/beacon_chain.proto

```diff
@@ -253,6 +253,13 @@ service BeaconChain {
             get: "/eth/v1alpha1/beacon/slashings/proposer/submit"
         };
     }
+
+    // Returns a list of validators individual vote status of a given epoch.
+    rpc GetIndividualVotes(IndividualVotesRequest) returns (IndividualVotesRespond) {
+        option (google.api.http) = {
+            get: "/eth/v1alpha1/debug/individual_votes"
```

Comment:
Please fix path. 

```suggestion
            get: "/eth/v1alpha1/beacon/individual_votes"
```

If "beacon" is not the appropriate path then please consider a different service for this method.